tests/v2/codecoverage/package/Dockerfile

```diff
@@ -56,7 +56,7 @@ ENV CATTLE_SYSTEM_UPGRADE_CONTROLLER_CHART_VERSION 101.0.0+up0.3.3
 ENV CATTLE_FLEET_MIN_VERSION=102.0.0+up0.6.0-rc.5
 # Deprecated.
 ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=''
-ENV CATTLE_RANCHER_WEBHOOK_VERSION=2.0.2+up0.3.2-rc21
+ENV CATTLE_RANCHER_WEBHOOK_VERSION=2.0.3+up0.3.3
```

Comment:
Ideally this should be the same RC as the main file is on, but I think that this works for now